### DISCUSSION TITLE
Compression ratio is slightly worse than in Pako

### DISCUSSION THREAD

**Discussion by mimi89999** [101arrowz/fflate#255] [General]
Hello,

I took the `dist/zxcvbn-ts.js` file from the https://www.npmjs.com/package/@zxcvbn-ts/language-pl/v/4.0.0-beta.0 and compressed it with fflate and Pako with level 9.
With fflate I got: zxcvbn-ts.js compressed from 821.1 kB to 374.3 kB
With Pako I got: zxcvbn-ts.js compressed from 821.1 kB to 352.0 kB

That's a difference of about 6%. It's small, but still noticeable.

**Comment by 101arrowz** [comment]
This is probably expected behavior given differences in the algorithm implementation between Pako and `fflate`. You are right that 6% is larger than usual though, it's worth looking into why this is (once I can find more time to do maintenance on this project!)
